modules/library/main/src/main/java/org/geotools/renderer/crs/WrappingProjectionHandler.java

```diff
@@ -49,6 +49,12 @@ public class WrappingProjectionHandler extends ProjectionHandler {
 
     private static final Object LARGE_EARTH_OBJECT = new Object();
 
+    /** The user data key to indicate that the geometry was pre-flipped */
+    protected static final String PREFLIPPED_OBJECT = "PRE-FLIPPED";
+
+    /** The tolerance to consider a geometry without touching both datelines */
+    public static final double DATELINE_PROXIMITY_TOLERANCE = 1.99572;
```

Comment:
Comment that it was obtained by trial and error.

Reply:
added